StandardConfig/production/run_standard_workflow.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+set -euo pipefail
+
+RUN_THE_THINGS=ON
+
+function run_cmd() {
+  local logfile=${1}
+  shift
+  echo $@
+  if [ "X${RUN_THE_THINGS}" = "XON" ]; then
+    $@ > ${logfile} 2>&1
+  fi
+}
+
+function clear_outputs() {
+
+  local output_files=( bbudsc_3evt_AIDA.root \
+    bbudsc_3evt_DST.slcio \
+    bbudsc_3evt_LCTuple.root \
+    bbudsc_3evt_PfoAnalysis.root \
+    bbudsc_3evt_REC.slcio \
+    bbudsc_3evt_SIM.slcio \
+    MarlinStdRecoParsed.xml \
+    ddsim.out \
+    marlin.out \
+    lctuple.out )
+
+  for file in $output_files; do
+    [ -f ${file} ] && rm ${file} || true
+  done
+}
+
+function prepare_lcfiweights() {
+  local lcfiweights=$(pwd)/../../LCFIPlusConfig/lcfiweights
+  local weights=4q250_ZZ_v4_p00_ildl5
+  if [ ! -d ${lcfiweights}/${weights} ]; then
+    echo "Unpacking LCFIPlus weights for test"
+    tar -xzf ${lcfiweights}/${weights}.tar.gz
```

Comment:
... these are exactly the commands we should add to the README.md

Reply:
Added a few sentences to the README